docs/pages/preview/responsive.tsx

```diff
@@ -269,7 +265,7 @@ export default function ResponsivePage() {
 								</Heading>
 								<Text>There was an error loading the preview.</Text>
 							</Stack>
-							<ButtonLink href={referrerLink}>Back to documentation</ButtonLink>
+							<ButtonLink href={referrerLink}>{referrerLabel}</ButtonLink>
```

Comment:
Shouldn't this be this?
```suggestion
							<ButtonLink href={referrerLink}>Back to {referrerLabel}</ButtonLink>
```